### PR TITLE
feat(appeals): add rows to s78 appellant case for infrastructure levy adopted date and expected date

### PR DESCRIPTION
**Pull request by james-hill-kc**
## Describe your changes

add rows to s78 appellant case for infrastructure levy adopted date and expected date

## Issue ticket number and link

A2-1286

## Type of change 🧩

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update
- [ ] Other (please explain in the description section above)

## Checklist before requesting a review

- [ ] I have performed a self-review of my own code
- [ ] I have double checked this work does not include any hardcoded secrets or passwords
- [ ] I have made corresponding changes to the documentation
- [ ] I have provided details on how I have tested my code
- [ ] I have referenced the ticket number above
- [ ] I have provided a description of the ticket
- [ ] I have included unit tests to cover any testable code changes

### CODE REVIEW

File: appeals/web/src/server/appeals/appeal-details/lpa-questionnaire/infrastructure-levy-expected-date/infrastructure-levy-expected-date.controller.js [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+import * as mapper from './infrastructure-levy-expected-date.mapper.js';
+import * as service from './infrastructure-levy-expected-date.service.js';
+
+/**
+ * @param {import('@pins/express/types/express.js').Request} request
+ * @param {import('@pins/express/types/express.js').RenderedResponse<any, any, Number>} response
+ */
+export const getChangeInfrastructureLevyExpectedDate = async (request, response) => {
+	return renderChangeInfrastructureLevyExpectedDate(request, response);
+};
```
Comment (Si3rr4wow) [inline appeals/web/src/server/appeals/appeal-details/lpa-questionnaire/infrastructure-levy-expected-date/infrastructure-levy-expected-date.controller.js:15]: `export const getChangeInfrastructureLevyExpectedDate = renderChangeInfrastructureLevyExpectedDate`

This is just a renaming I think?

File: appeals/web/src/server/lib/mappers/components/date.js [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+ * @param {string} options.text
+ * @param {string|null} [options.value]
+ * @param {string} [options.defaultText]
+ * @param {string} options.link
+ * @param {boolean} [options.addCyAttribute]
+ * @param {boolean} options.editable
+ * @param {string} [options.classes]
+ * @returns {Instructions}
+ */
+export function dateSummaryListItem({
```
Comment (Si3rr4wow) [inline appeals/web/src/server/lib/mappers/components/date.js:17]: Love that we're seeing more of these appear! I've just put in an instructions dir in this components dir, would you mind moving this into there please?

**Review by Si3rr4wow — APPROVED** [review]
Love what you're doing in here, looking very clean. Just a couple of suggested changes

File: appeals/web/src/server/appeals/appeal-details/lpa-questionnaire/infrastructure-levy-expected-date/infrastructure-levy-expected-date.controller.js (hunk 1 above)
Comment (james-hill-kc) [reply appeals/web/src/server/appeals/appeal-details/lpa-questionnaire/infrastructure-levy-expected-date/infrastructure-levy-expected-date.controller.js]: Thanks, updated!

File: appeals/web/src/server/lib/mappers/components/date.js (hunk 2 above)
Comment (james-hill-kc) [reply appeals/web/src/server/lib/mappers/components/date.js]: Done - sorry, I didn't think to update this after I rebased main into my branch! It's nice to have the separation between instructions and page components too :)

File: appeals/web/src/server/lib/mappers/components/date.js [hunk 3]
```diff
@@ -0,0 +1,107 @@
...
+ *
+ * @param {Object} params
+ * @param {string} params.name
+ * @param {string} [params.id]
+ * @param {string} [params.namePrefix]
+ * @param {DayMonthYearHourMinute|null} [params.value]
+ * @param {string} [params.legendText]
+ * @returns {PageComponent}
+ */
+export function dateInput({ name, id, namePrefix, value, legendText }) {
```
[the remark this answers is not in the document]
Comment (james-hill-kc) [reply appeals/web/src/server/lib/mappers/components/date.js]: done!

**Review by Si3rr4wow — APPROVED** [review]

**Pull request merged**